packages/teams-js/src/public/app/app.ts

```diff
@@ -771,6 +778,24 @@ export function registerHostToAppPerformanceMetricsHandler(handler: HostToAppPer
   Handlers.registerHostToAppPerformanceMetricsHandler(handler);
 }
 
+/**
+ * @beta
+ * Registers a function to handle memory metrics heartbeat sent from the host periodically.
+ *
+ * @remarks
+ * Only one handler can be registered at a time. A subsequent registration replaces an existing registration.
+ *
+ * @param handler - The handler to invoke each time memory metrics heartbeat is received from the host.
+ */
+export function registerHostMemoryMetricsHandler(handler: HostMemoryMetricsHandler): void {
```

Comment:
How will apps know if this API is supported? Should we move it under a separate namespace and its own capability - e.g. app.memory.registerHostMemoryMetricsHandler? That way an app can call the isSupported function under that namespace before trying to use the API (or get an error when it tries to call it)?